src/XFExpandableListView/Controls/ExpandableListView.cs

```diff
@@ -206,16 +209,19 @@ public virtual async Task Expand(int position)
             /* Hard operations must not affect the UI Thread */
             await Task.Run(() =>
             {
-                foreach (var item in expandableGroup)
+                Device.BeginInvokeOnMainThread(() =>
```

Comment:
I think there is no point on creating a new thread which will executes on the main thread. Just remove the task.run